platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/Capture.kt

```diff
@@ -83,6 +83,8 @@ object Capture {
             dateProvider: DateProvider? = null,
             apiUrl: HttpUrl = defaultCaptureApiUrl,
         ) {
+            // TODO Adding a no-op for testing, please remove after confirming everything is good
+            val test = true
```

Comment:
this might break detekt, plus I agree with your statement about not polluting the repo with these and TODOS since they will go into `main`. How about something inconsequential that can actually stay in teh codebase? Something like a variable rename for example

Reply:
Sounds good to me. Let me quickly fix 👍

Reply:
Done 👉 https://github.com/bitdriftlabs/capture-sdk/pull/32/commits/ecd11acfd5c74888a7285fef6753700982d81bf5